BUILD

```diff
@@ -965,13 +964,14 @@ grpc_cc_library(
     ],
     language = "c++",
     public_hdrs = GPR_PUBLIC_HDRS,
-    tags = ["nofixdeps"],
-    visibility = ["@grpc:alt_gpr_base_legacy"],
+    tags = [
+        "avoid_dep",
```

Comment:
Suspect we should remove the avoid_dep here... could you try and see whether fix_build_deps stays rational?

Reply:
I tried, and I did not find anything change because of this tag.